### DISCUSSION TITLE
Support for Chargebee

### DISCUSSION THREAD

**Discussion by pmackay** [pay-rails/pay#917] [Ideas]
I would be interested to know if there has been any consideration for supporting [Chargebee](https://www.chargebee.com/gb/) as a payment processor?

**Comment by excid3** [comment]
First time anyone's mentioned it. You could build it as a plugin for Pay.

**Comment by pathe-kantox** [comment]
How would a pay plugin looks like ?

**Reply by excid3** [reply]
Define the classes and methods like the other payment processors do.
